dcpy/connectors/edm/publishing.py

```diff
@@ -202,22 +204,53 @@ def publish(
     keep_draft: bool = True,
     max_files: int = s3.MAX_FILE_COUNT,
     latest: bool = False,
```

Comment:
I'm curious whether you think we should keep this flag? Ie is there a case where we'd actually want to publish a later version, and _not_ update the latest folder?

Reply:
Rare case, but if we publish 24v2 and need to patch some horrible error in 24v1.1? Say a completely nonsense bbl with 100000 units that we want to correct for posterity

Reply:
@fvankrieken Sorry, to be clear, what I mean is if we infer that something is the latest or not programmatically, why has this optional at all? So in the case, we'd detect that 24v1.1 isn't latest (which, for us really means latest version, as opposed to latest built) and just not push it to the `latest` folder.

Reply:
OH great point. Sounds good to me

Reply:
My thoughts around this: suppose we manually entered incorrect *future* version in the GHA, resulting in unwanted, updated `latest` version. Then, as far as I know, we don't have means to trace what version we had in latest before. Ideas?

Reply:
Related to "overwrite if exists" comments - I think just having some amount of functionality to either
- manually specify a published version as latest (just overwrite "latest" folder)
- reset "latest" (look at published versions, programmatically determine latest, update latest folder)

So just keeping functionality so that what we do to "latest" is somewhat flexible (since we overwrite it all the time and can't get around its mutability), but not having that flexibility around published versions

Reply:
Per discussion, keep as is for now.